client/signup/main.jsx

```diff
@@ -593,7 +593,7 @@ export default connect(
 		);
 		return {
 			domainsWithPlansOnly: getCurrentUser( state )
```

Comment:
Would it make a difference if we set this true always?

```suggestion
			domainsWithPlansOnly: true,
```

Reply:
Yes, it's expected from current exempt from DWPO users (which were pulled into "exempt from PWPO" bucket). That's how it was working before all these changes and I'd prefer to preserve that to avoid too many variables and changes.